packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -22,18 +22,39 @@ const { OrgAuthInfo, channelService } = sfdxCoreExports;
 // This should be exported from soql-builder-ui
 export interface SoqlEditorEvent {
   type: string;
-  message?: string;
+  payload?: string | string[] | ToolingModelJson;
 }
 
 // This should be shared with soql-builder-ui
 export enum MessageType {
-  ACTIVATED = 'activated',
-  QUERY = 'query',
+  UI_ACTIVATED = 'ui_activated',
+  UI_SOQL_CHANGED = 'ui_soql_changed',
   SOBJECT_METADATA_REQUEST = 'sobject_metadata_request',
   SOBJECT_METADATA_RESPONSE = 'sobject_metadata_response',
   SOBJECTS_REQUEST = 'sobjects_request',
   SOBJECTS_RESPONSE = 'sobjects_response',
-  UPDATE = 'update'
+  TEXT_SOQL_CHANGED = 'text_soql_changed'
+}
+
+// TODO: move to shared module
```

Comment:
@dobladez - Where do you think we should move them and why?

Reply:
Hmm... I was under the impression that Org connection handling was something we needed somewhere else. 
As for `MessageType` and `SoqlEditorEvent`, I'll probably make the change (to include them from `sql-builder-ui`) as part of my next work item).

Reply:
You could update `getConnection` in the OrgAuthInfo util to accept either a `usernameOrAlias` or `undefined`. If it's undefined it'll just use the default one like you're doing here.